README.md

```diff
@@ -10,24 +10,34 @@ The CLI builds a Docker container image locally that’s similar to a MWAA produ
 
 ```text
 dags/
+  example_dag_with_custom_ssh_plugin.py
```

Comment:
Hi @noharafat! I was looking forward for this update, sorry to intrude the PR hahaha 

I saw the folder structure for the examples, wouldn't maybe be more interesting for the examples stay inside an "example" folder? and the plugins inside the DAG example also, instead of the root, might be more easy to understand the semantics of each thing. When I saw plugins I thought it was something specific for the MWAA to use to run the service

Not wanting to be annoying (but already being a little bito), if doesn't make sense it's ok, thanks a whole LOT for the contribution :)